discovery-provider/.test.env

```diff
@@ -0,0 +1,3 @@
+# TODO: dummy ganache keys for local setup; should wire with dynamically generated keys
```

Comment:
does this block CI execution? jw because we should really try to avoid committing even dummy private keys - if these aren't set does that break things?

Reply:
We do this already in .env -- and we need it or else the config.py doesn't load.

I think cleaning up the env vars is needed, but a separate task. the Config.py script is really hard to mock because it runs before pytest's earliest hookimpl can even run (tried this)

Reply:
Rather i should say, there's no way to run pytest without config.py running and we can't run unit tests w/o pytest running.

Reply:
Re-using the entire .env for the test case here doesn't work b/c it messes up the env vars while running on linux. I think there's a world in which we can rework the env var set-up entirely, so we just don't have any .env files, but that will require programmatically generating wallet keys (i.e. the TODO)

Reply:
yeah it should never have been allowed to go into `.env`, i guess it's okay since this pattern is already broken but this is pretty wrong. @dmanjunath can we make an item to remove all private keys in the repo?